src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics.Test/ScenarioTests/AdlaTestsBase.cs

```diff
@@ -64,6 +65,8 @@ public class AdlaTestsBase : RMTestBase
 
         public DataLakeAnalyticsCatalogManagementClient DataLakeAnalyticsCatalogManagementClient { get; private set; }
 
+        public GraphRbacManagementClient GraphRbacManagementClient { get; private set; }
```

Comment:
@idear1203 nit: we should remove this client since we don't use it anywhere in the cmdlet code